common/patches/ss_define.h

```diff
@@ -68,7 +68,7 @@
 #undef OUT
 #define OUT(x) eq->x = emu->x;
 #define OUT_str(x) \
-	strncpy(eq->x, emu->x, sizeof(eq->x)); \
+	strn0cpy(eq->x, emu->x, sizeof(eq->x)); \
```

Comment:
This one should be reverted since the macro already handled adding null in its own way and we probably don't want to change how this works in this patch